lib/Extension/IndexerExtension.php

```diff
@@ -94,7 +94,7 @@ public function configure(Resolver $schema)
         $schema->setDescriptions([
             self::PARAM_ENABLED_WATCHERS => 'List of allowed watchers. The first watcher that supports the current system will be used',
             self::PARAM_INDEX_PATH => 'Path where the index should be saved',
-            self::PARAM_STUB_PATHS => 'Paths to folders where code stubs are located',
+            self::PARAM_STUB_PATHS => "Paths to external folders to index\n\nThey will be indexed only once, if you want to take any changes into account you will have to reindex your project manually.",
```

Comment:
Should we have `\n\n` here? This could be rendered as HTML and this would be ignored. In general all the option documentations are a single line.

Reply:
I guess it's fine

Reply:
OK, I'll remove it but it start to be a bit long